.github/workflows/uncrustify.yml

```diff
@@ -0,0 +1,49 @@
+name: Uncrustify the source code
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  Uncrustify:
+    name: Run_Uncrustify
+    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/bot run uncrustify' }}
+    runs-on: ubuntu-18.04
```

Comment:
18.04 is getting a bit old. Will this run on ubuntu-latest?

Reply:
18.04 comes with a certain version of uncrustify. That is the one we use. That is why we stuck with using Ubuntu 18.04.